modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-0_{context}"]
+= Release notes for {gitops-title} 1.4.0
+
+{gitops-title} 1.4.0 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
```

Comment:
I thought we only publish 1.4.0 to OCP 4.7, 4.8, 4.9 and (once it is released) 4.10. OCP 4.6 is out of scope for GitOps 1.4.0 release.

Reply:
cc @wtam2018

Reply:
Removed 4.6. Thanks.